src/contract/assembled_transaction.ts

```diff
@@ -800,33 +803,40 @@ export class AssembledTransaction<T> {
      * Sign all auth entries for this account. Default: the account that
      * constructed the transaction
      */
-    publicKey?: string;
+    address?: string;
```

Comment:
This is the breaking change, right? We need to document this boldly in the changelog :pray: